configure.ac

```diff
@@ -355,6 +355,24 @@ if test x$use_sanitizers != x; then
     ]],[[]])])
 fi
 
+dnl GCC<9.2 reports a redundant -Wsuggest-override warning on a 'final' method.
+dnl See https://gcc.gnu.org/bugzilla/show_bug.cgi?id=78010
+AC_MSG_CHECKING(for GCC compiler version < 9.2)
```

Comment:
nit: just "... GCC version ..." is ok too (remove "compiler" because "GCC" means "GNU Compiler Collection")

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/16710#issuecomment-627245385).